index.php

```diff
@@ -263,8 +277,9 @@
 	        <p>
 		    <center>
 					<img src="https://i.imgur.com/nPxjS4J.gif" />
-			    		<p style="font-family:courier; font-size:60%; color: white;">Cursed Image &copy; Maria 2017</p>
-			</center>
+					<p style="font-family:courier; font-size:60%;">Cursed Image &copy; Maria 2017</p>
+					<iframe width="1280" height="720" src="https://www.youtube.com/embed/96Mbt36ayZ8" frameborder="0" allowfullscreen></iframe>

```

Comment:
Rule #1 of pleasing me: tags must end either on the same line, or with the same indentation a few lines down